daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,29 @@ enum RollOverState {
     None,
 }
 
+pub struct AutoRolloverParams {
+    settlement_interval: Duration,
+    /// Defines the minimum duration a CFD needs to be open for until a roll-over is triggered.
+    max_open_until_rollover: Duration,
+}
+
+impl AutoRolloverParams {
+    pub fn new(rollover_interval: Duration, settlement_interval: Duration) -> Result<Self> {
+        Ok(Self {
+            settlement_interval,
+            max_open_until_rollover: rollover_interval,
+        })
+    }
+
+    // Two conditions need to be bet for a roll over to be triggered.
+    // The cfd should not of run longer that the settlement interval.
+    // The cfd should of lived longer than the rollover interval.
+    pub fn is_cfd_due_for_rollover(&self, elapsed_since_cfd_opened: Duration) -> bool {
+        self.settlement_interval > elapsed_since_cfd_opened
```

Comment:
Just to make sure I understand correct.

The `settlement_interval` is a constant set to `24` hours. This allows rollover if that constant is greater than the time elapsed since the CFD was opened. 
So, if `elapsed_since_cfd_opened` is `25` hours we won't roll over.

Reply:
yep